gwcli/gateway.py

```diff
@@ -417,6 +417,7 @@ def __init__(self, target_iqn, parent):
         config = self.parent.parent._get_config()
         if not config:
             self.logger.error("Unable to refresh local config")
+            return
```

Comment:
I'd rather throw something here instead of partially initializing the object. That seems like it would just allow other errors to sneak in.

Reply:
Okay, will fix it.